docs/user/security/secure-saved-objects.asciidoc

```diff
@@ -0,0 +1,90 @@
+[role="xpack"]
+[[xpack-security-secure-saved-objects]]
+=== Secure Saved Objects
```

Comment:
__note:__ didn't want to be too specific and say `Encrypt`, used more generic `Secure` word....